checker/tests/index/UnaryOperationParsedIncorrectly.java

```diff
@@ -0,0 +1,12 @@
+import org.checkerframework.checker.index.qual.LessThan;
+
+class Issue {
```

Comment:
For test cases, the class name should match the filename and should be public.  This is because all the java files in test directory are compiled together so it's easy to make sure that two test cases don't overlap.